code/game/objects/structures/barsign.dm

```diff
@@ -42,14 +42,15 @@ var/list/barsigns = list()
 //Custom Barsign Var shit
 	var/obj/effect/overlay/custom_barsign/ass = null
 	var/list/sound_selection = list("Nothing" = null,
-									"Rooster" = 'sound/misc/6amRooster.wav',
-									"Wolf" = 'sound/misc/6pmWolf.wav',
-									"Male Scream" = 'sound/misc/malescream5.ogg',
-									"Female Scream" = 'sound/misc/femalescream5.ogg',
-									"Vox Scream" = 'sound/misc/shriek1.ogg',
-									"Bike Horn" = 'sound/items/bikehorn.ogg'
+									"bass_drum_1"	= 'sound/items/bsign_1.ogg',
```

Comment:
I think it's a good idea to rename the audio files to what they represent rather than what they're used for

Reply:
Yeah sure